while-loops/while-loops.md

```diff
@@ -0,0 +1,257 @@
+# While Loops
+
+## Learning Goals
+
+* Practice using while loops to loop while a conditional is true 
+* Apply a counter variable while using while loops
+
+## Introduction
+
+Imagine writing software that creates digital group greeting cards. Before the user can sign a digital group greeting card, they need to type in the specific password `AdaLovelaceCodesIt` . If they don't type in the correct password, then the app should ask them to type in the password again, and again, and again, until the user leaves or types in `AdaLovelaceCodesIt` .
+
+How can we solve this problem using Python code?
+
+for loops allow programmers to iterate over lists. Sometimes, we need to repeat lines of code, but not because we are iterating through a list. We can explore the while loop as an alternative!
```

Comment:
```suggestion
`for` loops allow programmers to iterate over lists. Sometimes, we need to repeat lines of code, but not because we are traversing (iterating through) a list. We can explore the `while` loop as an alternative!
```

Added `for` and `while` formatting -- I will go through these lessons and apply this formatting lesson-wide afterwards

Went with "traversing" as the main verb, and "iterating through" as a parenthetical!

LMK what you think! Closing this suggestion not to stop conversation, but to help me move forward.